### ISSUE TITLE
AUR packages not installing in the correct order

### ISSUE THREAD

**Issue by chrjen** [Jguer/yay#853]
#### Affected Version
yay v9.0.1 - libalpm v11.0.1

#### Issue

[toilet](https://aur.archlinux.org/packages/toilet/) and [toilet-fonts](https://aur.archlinux.org/packages/toilet-fonts/) does not install in the correct order when installed together.

#### Steps to reproduce

0. Don't have either of the packages installed.
1. `$ yay toilet`
2. Input relevant number, 1 and 2 for me.

#### Output

```bash
[...]
:: Parsing SRCINFO (1/2): toilet-fonts
:: Parsing SRCINFO (2/2): toilet
==> Lager pakke: toilet-fonts 1.0-2 (lø. 05. jan. 2019 kl. 21.44 +0100)
==> Henter kilder...
  -> Fant ours.tar.gz
  -> Fant contributed.tar.gz
  -> Fant international.tar.gz
  -> Fant ms-dos.tar.gz
==> Validerer source filer med sha256sums...
    ours.tar.gz ... Godkjent
    contributed.tar.gz ... Godkjent
    international.tar.gz ... Godkjent
    ms-dos.tar.gz ... Godkjent
==> Rydder...
==> Lager pakke: toilet 0.3.r155.3eb9d58-1 (lø. 05. jan. 2019 kl. 21.44 +0100)
==> Henter kilder...
  -> Fant toilet-0.3.r155.3eb9d58.tar.gz
==> Validerer source filer med sha256sums...
    toilet-0.3.r155.3eb9d58.tar.gz ... Godkjent
==> Rydder...
==> Lager pakke: toilet-fonts 1.0-2 (lø. 05. jan. 2019 kl. 21.44 +0100)
==> Kontrollerer avhengigheter som trengs ved kjøring...
==> Manglende avhengigheter:
  -> toilet
==> Kontrollerer avhengigheter som trengs ved bygging...
==> FEIL: Fant ikke avhengighetene.
Error making: toilet-fonts
```

Basically it tries to install toilet-fonts before toilet which fails because toilet-fonts is dependent on toilet.

Installing **only** toilet-fonts works and will install toilet first. Likewise manually installing toilet first, when using yay again to install toilet-fonts works.

**Comment by Morganamilo** [comment]
Assuming this was fixed by #866. If not feel free to repoen.
